mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/social/ChessMatch.kt

```diff
@@ -0,0 +1,20 @@
+package ch.epfl.sdp.mobile.ui.features.social
+
+class ChessMatch(val adversary: String, val matchResult: MatchResult, val numberOfMoves: Int)

```

Comment:
A `data class` is preferred to handle equality checks etc.

```suggestion
data class ChessMatch(val adversary: String, val matchResult: MatchResult, val numberOfMoves: Int)

```